package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "helsinki-utils",
+  "version": "0.0.0",
+  "description": "Common code used by various City of Helsinki frontends",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/City-of-Helsinki/helsinki-utils-js.git"
+  },
+  "author": "",
+  "license": "ISC",
```

Comment:
Probably doesn't matter, but I think almost every project has license set to MIT

Reply:
Good point actually 👍